docs/application/dotnet/guides/nui/nui-components/Slider.md

```diff
@@ -0,0 +1,82 @@
+# Slider
+A slider lets users select a value from a continuous or discrete range of values by moving the slider thumb.
+
+![Slider](./media/slider.png) ![Slider](./media/slider2.png)
+
+## Overview
+Slider is a kind of common component and lets users select a value from a continuous or discrete range of values by moving the slider thumb.
+
+- Use a slider to adjust the steps or strength of settings, such as the brightness and contrast.
+- Sliders are classified into a horizontal type, vertical type depending on their orientation and the number of sliders that are adjusted simultaneously.
+
+## Create with property
+1. Create Slider by default constructor
+
+~~~{.cs}
+utilityBasicSlider = new Slider();
+~~~
+
+2. Set slider property
+
+~~~{.cs}
+utilityBasicSlider.TrackThickness = 4;
+utilityBasicSlider.BgTrackColor = new Color(0, 0, 0, 0.1f);
+utilityBasicSlider.SlidedTrackColor = new Color(0.05f, 0.63f, 0.9f, 1);
+utilityBasicSlider.ThumbImageURLSelector = new StringSelector
+{
+    Normal = "controller_btn_slide_handler_normal.png",
+    Pressed = "controller_btn_slide_handler_press.png",
+};
+utilityBasicSlider.ThumbSize = new Size(60, 60);
+utilityBasicSlider.Direction = Slider.DirectionType.Horizontal;
+root.Add(utilityBasicSlider);

```

Comment:
It'd be better to move this `root.Add` to the end of the code like other tutorials.
For beginners, it looks like having special meaning to have this line in the middle of setting properties.

Reply:
Done